lib/releaselib.sh

```diff
@@ -952,7 +952,7 @@ release::docker::release () {
   cd "$release_images" || return 1
   echo --  *)
   for arch in "${arches[@]}"; do
-    for tarfile in "$release_images/$arch/"*.tar; do
+    for tarfile in "$release_images/$arch"/*.tar; do
```

Comment:
looks like this would work

```
[dims@dims-vmware-laptop 11:12] ~ ⟩ cd junk/
[dims@dims-vmware-laptop 11:12] ~/junk ⟩ mkdir release_images
[dims@dims-vmware-laptop 11:12] ~/junk ⟩ cd release_images/
[dims@dims-vmware-laptop 11:12] ~/junk/release_images ⟩ mkdir arm
[dims@dims-vmware-laptop 11:12] ~/junk/release_images ⟩ mkdir amd
[dims@dims-vmware-laptop 11:12] ~/junk/release_images ⟩ mkdir ppc
[dims@dims-vmware-laptop 11:12] ~/junk/release_images ⟩ touch arm/1.tar
[dims@dims-vmware-laptop 11:12] ~/junk/release_images ⟩ touch amd/2.tar
[dims@dims-vmware-laptop 11:12] ~/junk/release_images ⟩ touch ppc/3.tar
[dims@dims-vmware-laptop 11:13] ~/junk/release_images ⟩ cd ..
[dims@dims-vmware-laptop 11:13] ~/junk ⟩ release_images=release_images
[dims@dims-vmware-laptop 11:13] ~/junk ⟩ arches=(arm amd ppc)
[dims@dims-vmware-laptop 11:13] ~/junk ⟩ for arch in "${arches[@]}"; do
>   echo $arch
> done
arm
amd
ppc
[dims@dims-vmware-laptop 11:13] ~/junk ⟩ for arch in "${arches[@]}"; do
>   for tarfile in "$release_images/$arch"/*.tar; do
>     echo $tarfile
>   done
> done
release_images/arm/1.tar
release_images/amd/2.tar
release_images/ppc/3.tar
```